doc/release/release_notes.rst

```diff
@@ -7,6 +7,13 @@
 Release Notes
 *************
 
+.. release:: upcoming
+
+    .. change:: new
+
+    3ds Max integration.
```

Comment:
@henriknorin-ftrack 

something more like: 

`Add 3dsmax definition and plugins `

might explain better what is this about

Reply:
Changing